docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -316,6 +317,7 @@ Json Parameters:
           `json-file` logging driver.
     -   **CgroupParent** - Path to `cgroups` under which the container's `cgroup` is created. If the path is not absolute, the path is considered to be relative to the `cgroups` path of the init process. Cgroups are created if they do not already exist.
     -   **VolumeDriver** - Driver that this container users to mount volumes.
+    -   **ShmSize** - Size of /dev/shm. The unit is number of byte. When not specified, default value is 64MB.
```

Comment:
Thank you for pointing it out.
Because remote_api is passed the value with int64, the unit is unnecessary. 
In the command line, because the argument that the user specified is parsed, the unit can be specified.